aspnetcore/grpc/browser.md

```diff
@@ -20,37 +20,21 @@ It is not possible to call a HTTP/2 gRPC service from a browser-based app. [gRPC
 
 ## Configure gRPC-Web in ASP.NET Core
 
-gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services, the only modification is startup configuration.
+gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services. The only modification is startup configuration.
 
 To enable gRPC-Web with an ASP.NET Core gRPC service:
 
 1. Add a reference to the [Grpc.AspNetCore.Web](https://www.nuget.org/packages/Grpc.AspNetCore.Web) package.
```

Comment:
Use *, not 1, 2
You don't need to do those steps in that order. Even if you do, we prefer *.  When you're forced to use numbers, use 1. each time. 
```
1.
1.
1.
```
The build system will number them. Updates don't need to be edited.